content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,136 @@
+
+    {{< note >}}
+
+    Control plane components run as static pod manifests, managed by the kubelet. Components are not disrupted by in-place updates of CA certificates until someone kills or restarts an already running pod.
+
+    {{< /note >}}
+
+1. Get new certificates and private keys signed from the new CA for `apiserver.crt`, `apiserver-kubelet-client.crt`, and `front-proxy-client.crt` and replace them in the Kubernetes certificates directory.
+
+    To generate certificates and private keys for your cluster using openssl, see "[openssl](/docs/concepts/cluster-administration/certificates/#openssl)". 
+    
+    You can also use [cfssl](/docs/concepts/cluster-administration/certificates/#cfssl).
+
+
+1. Update the config map `cluster-info` in the kube-public namespace.
+
+    ```shell
+    base64_encoded_ca=$(/usr/bin/base64 /etc/kubernetes/pki/ca.crt)
+
+    kubectl get cm/cluster-info --namespace kube-public -o yaml | \
+    /bin/sed "s/\(certificate-authority-data:\).*/\1 ${base64_encoded_ca}" | \
+    kubectl apply -f -
+    ```
+
+1. Update all service account tokens with the new CA certificates.
+    ```shell
+    base64_encoded_ca=$(/usr/bin/base64 /etc/kubernetes/pki/ca.crt)
+
+    for namespace in $(kubectl get ns --no-headers); do
+        for token in $(kubectl get secrets --namespace $namespace --field-selector type=kubernetes.io/service-account-token -o name); do
+            kubectl get $token --namespace $namespace -o yaml | \
+            /bin/sed "s/\(ca.crt:\).*/\1 ${base64_encoded_ca}" | \
+            kubectl apply -f -
+        done
+    done
+
+    ```
+
+1. Update certificates for user accounts.
+
+    For information about creating certificates for individual user accounts, see "[Configure certificate for user accounts](/docs/setup/best-practices/certificates/#configure-certificates-for-user-accounts)".
+
+    Update `client-certificate-data` and `client-key-data` respectively with the contents of certificates and private keys generated in their respective conf files, along with the kubelet client file in`/var/lib/kubelet/kubelet-client-current.pem`.
+
+1. Update `kubelet.conf`, the `kubelet` certificate, and the private key on all worker nodes as well.
+
+1. Update the `certificate-authority-data` section in all `admin.conf`, `controller-manager.conf`,  `scheduler.conf`, and `kubelet.conf` files with base64 encoded CA certificate data.
+
+    Example:
+    ```shell
+    base64_encoded_ca=$(/usr/bin/base64 /etc/kubernetes/pki/ca.crt)
+
+    /bin/sed -i "s/\(certificate-authority-data:\).*/\1 ${base64_encoded_ca}" /path/to/conf/files
+    ```
+
+1. Follow these steps on all control plane and worker nodes.
+
+    a. Stop the kubelet service on all control plane and worker nodes.
+
+    a. Kill the static pod's containers manually on control plane nodes.
+        ```shell
```

Comment:
I'd add a comment to tell the reader that this command assumes they are using Docker as the container runtime on their control plane.

Reply:
done